library/Playground/HTTP/src/Juvix/Playground/HTTP/Routes.hs

```diff
@@ -27,56 +33,21 @@ import qualified Juvix.Library.NameSymbol as NameSymbol
 import qualified Juvix.Pipeline as Pipeline
 import qualified Juvix.Sexp as Sexp
 import Servant
+import Text.Pretty.Simple (pShowNoColor)
 
 type Result a = Feedback.Feedback [] [Char] a
 
--- TODO: Include backend details
-data Req = Req {reqBody :: Text}
-  deriving (Eq, Show, Generic, A.ToJSON, A.FromJSON)
-
-type ToML =
-  Description "Convert Juvix source code to AST"
-    :> ReqBody '[JSON] Req
-    :> Post '[JSON] (Result [(NameSymbol.T, [Frontend.TopLevel])])
-
-type ToSexp =
-  Description "Convert AST to S-expression"
-    :> ReqBody '[JSON] [(NameSymbol.T, [Frontend.TopLevel])]
-    :> Post '[JSON] (Result (Context.T Sexp.T Sexp.T Sexp.T))
-
-type ToHR =
-  Description "Convert S-expression to HR"
-    :> ReqBody '[JSON] (Context.T Sexp.T Sexp.T Sexp.T)
-    :> Post '[JSON] (Result (Core.RawGlobals HR.T (Plonk.PrimTy Fr) (Plonk.PrimVal Fr)))
-
-type ToIR =
-  Description "Convert HR to IR"
-    :> ReqBody '[JSON] (Core.RawGlobals HR.T (Plonk.PrimTy Fr) (Plonk.PrimVal Fr))
-    :> Post '[JSON] (Result (Core.PatternMap Core.GlobalName, Core.RawGlobals IR.T (Plonk.PrimTy Fr) (Plonk.PrimVal Fr)))
+data Backend
+  = Plonk (Plonk.BPlonk Fr)
+  | LLVM LLVM.BLLVM
+  deriving (Eq, Show, Generic, A.ToJSON)
```

Comment:
Guessing the Michelson backend can't work here at all?

Reply:
Let me add it